patch-package
--- a/node_modules/react-d3-radar/lib/Radar.js
+++ b/node_modules/react-d3-radar/lib/Radar.js
@@ -73,7 +73,11 @@ function Radar(props) {
       domainMax = props.domainMax,
       style = props.style,
       onHover = props.onHover,
-      highlighted = props.highlighted;
+      highlighted = props.highlighted,
+      colors = props.colors,
+      numRings = props.numRings,
+      format = props.format,
+      fontSize = props.fontSize;
 
   var _convertData = convertData(props),
       allPoints = _convertData.allPoints,
@@ -86,12 +90,12 @@ function Radar(props) {
 
   var backgroundScale = scales[data.variables[0].key];
 
-  var colors = {};
-  (0, _utils.forEachArray)(allPoints, function (_ref3, idx) {
-    var setKey = _ref3.setKey;
+  //var colors = {};
+  //(0, _utils.forEachArray)(allPoints, function (_ref3, idx) {
+  //  var setKey = _ref3.setKey;
 
-    colors[setKey] = _d3Scale.schemeCategory10[idx];
-  });
+  //  colors[setKey] = _d3Scale.schemeCategory10[idx];
+  //});
 
   var _$partition = _lodash2.default.partition(allPoints, function (_ref4) {
     var setKey = _ref4.setKey;
@@ -117,6 +121,9 @@ function Radar(props) {
     radius: radius,
     highlightedPoint: highlightedPoints.length > 0 ? highlightedPoints[0] : null,
     regularPoints: regularPoints,
-    colors: colors
+    colors: colors,
+    numRings: numRings,
+    format: format,
+    fontSize: fontSize
   });
 }
\ No newline at end of file
--- a/node_modules/react-d3-radar/lib/RadarAxis.js
+++ b/node_modules/react-d3-radar/lib/RadarAxis.js
@@ -20,7 +20,7 @@ var defaultRadarAxisStyle = {
   fontSize: 10,
   fontFamily: 'sans-serif',
   textFill: 'black',
-  axisWidth: 2
+  axisWidth: 1
 };
 
 function RadarAxis(props) {
@@ -29,12 +29,14 @@ function RadarAxis(props) {
       domainMax = props.domainMax,
       label = props.label,
       color = props.color,
-      style = props.style;
+      style = props.style,
+      fontSize = props.fontSize,
+      index = props.index;
 
   var _defaultRadarAxisStyl = _extends({}, defaultRadarAxisStyle, { style: style }),
       axisOverreach = _defaultRadarAxisStyl.axisOverreach,
       labelOverreach = _defaultRadarAxisStyl.labelOverreach,
-      fontSize = _defaultRadarAxisStyl.fontSize,
+      //fontSize = _defaultRadarAxisStyl.fontSize,
       fontFamily = _defaultRadarAxisStyl.fontFamily,
       textFill = _defaultRadarAxisStyl.textFill,
       axisWidth = _defaultRadarAxisStyl.axisWidth;
@@ -55,6 +57,8 @@ function RadarAxis(props) {
     _react2.default.createElement(
       'text',
       {
+        id: 'text' + index,
+        className: 'text',
         x: scale(domainMax * labelOverreach) * xFactor,
         y: scale(domainMax * labelOverreach) * yFactor,
         fontSize: fontSize,
--- a/node_modules/react-d3-radar/lib/RadarCircle.js
+++ b/node_modules/react-d3-radar/lib/RadarCircle.js
@@ -24,11 +24,11 @@ var defaultCircleStyle = {
   selectedFillOpacity: 0.5,
   inactiveFillOpacity: 0.2,
   selectedStrokeOpacity: 1.0,
-  inactiveStrokeOpacity: 0.7,
+  inactiveStrokeOpacity: 1.0,
   pointRadius: 3,
   selectedPointFill: 'white',
   selectedPointOpacity: 1.0,
-  inactivePointOpacity: 0.7
+  inactivePointOpacity: 1.0
 };
 
 function RadarCircle(props) {
@@ -67,8 +67,9 @@ function RadarCircle(props) {
       stroke: color,
       strokeOpacity: isSelected ? selectedStrokeOpacity : inactiveStrokeOpacity
     }),
-    points.map(function (point) {
+    points.map(function (point, index) {
       return _react2.default.createElement('circle', {
+        className: 'circle circle' + index,
         key: point.key,
         r: pointRadius,
         fill: point.variableKey === selectedVariableKey ? selectedPointFill : color,
--- a/node_modules/react-d3-radar/lib/RadarWrapper.js
+++ b/node_modules/react-d3-radar/lib/RadarWrapper.js
@@ -120,20 +120,23 @@ var RadarWrapper = function (_Component) {
           highlightedPoint = _props2.highlightedPoint,
           regularPoints = _props2.regularPoints,
           backgroundScale = _props2.backgroundScale,
-          colors = _props2.colors;
+          colors = _props2.colors,
+          numRings = _props2.numRings,
+          format = _props2.format,
+          fontSize = _props2.fontSize;
 
       var diameter = radius * 2;
 
       var _defaultRadarStyle$st = _extends({}, defaultRadarStyle, style),
           axisColor = _defaultRadarStyle$st.axisColor,
-          ringColor = _defaultRadarStyle$st.ringColor,
-          numRings = _defaultRadarStyle$st.numRings;
+          ringColor = _defaultRadarStyle$st.ringColor;
 
       var innerHeight = height - padding * 2;
       var innerWidth = width - padding * 2;
 
       var ticks = backgroundScale.ticks(numRings).slice(1);
-      var tickFormat = backgroundScale.tickFormat(numRings);
+      var tickFormat = format;
+      //var tickFormat = backgroundScale.tickFormat(numRings);
 
       return _react2.default.createElement(
         'svg',
@@ -161,17 +164,19 @@ var RadarWrapper = function (_Component) {
               color: ringColor,
               format: tickFormat
             }),
-            variables.map(function (_ref2) {
+            variables.map(function (_ref2, index) {
               var key = _ref2.key,
                   label = _ref2.label;
 
               return _react2.default.createElement(_RadarAxis2.default, {
+                index: index,
                 key: key,
                 scale: scales[key],
                 offsetAngle: offsetAngles[key],
                 label: label,
                 domainMax: domainMax,
-                color: axisColor
+                color: axisColor,
+                fontSize: fontSize
               });
             }),
             regularPoints.map(function (_ref3) {
